CONTRIBUTING.md

```diff
@@ -59,6 +59,41 @@ of Windows.
 * Visual Studio 2017+ or Visual Studio Code
 * .NET Framework 4.6+
 
+### Visual Studio Code
+
+This repository contains example configuration for Visual Studio Code located
+under `.vscode.example`. You can copy it to `.vscode`.
+
+```sh
+cp -r .vscode.example .vscode
```

Comment:
This seems specific to VSCode on Linux/Mac (or Windows with Ubuntu mode).

Reply:
Works in Git Bash on Windows. I guess it would work in PowerShell as well